DigitalExcellence/dex-frontend
# added mobile styling to overlay

## Description

Added mobile styling exceptions to the edit page where needed to get around the hover-overlay.

_there's no sprint 3 board yet so I'll add this PR it it as soon as that's there + I'll add a changelog entry when the changlog has a part for the next release, as discussed with @DaveBouman  😄_

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to DeX on mobile or create mobile in devtools
2. Go to edit page and the images tab
3. See the image previews
4. Note that its now clear what will happen when you click on the icon or image-preview

## Link to issue

Closes: #584
